packages/cli/src/commands/build.ts

```diff
@@ -469,6 +469,8 @@ async function doBuild(
         )
       );
     } catch (err: any) {
+      output.prettyError(err);
```

Comment:
@TooTallNate Do we need this to conditionally print so we don't accidentally print twice in the build container?

Reply:
Looks like we can solve this in the build container 👍